yb-voyager/cmd/root.go

```diff
@@ -66,29 +66,46 @@ Refer to docs (https://docs.yugabyte.com/preview/migrate/) for more details like
 		if !shouldRunPersistentPreRun(cmd) {
 			return
 		}
-		validateExportDirFlag()
-		schemaDir = filepath.Join(exportDir, "schema")
-		if shouldLock(cmd) {
-			lockFPath := filepath.Join(exportDir, fmt.Sprintf(".%sLockfile.lck", GetCommandID(cmd)))
-			lockFile = lockfile.NewLockfile(lockFPath)
-			lockFile.Lock()
-		}
-		InitLogging(exportDir, cmd.Use == "status", GetCommandID(cmd))
-		startTime = time.Now()
-
-		if callhome.SendDiagnostics {
-			createCLIArgsString(cmd)
-			go sendCallhomePayloadAtIntervals()
-		}
 
-		log.Infof("Start time: %s\n", startTime)
-		if metaDBIsCreated(exportDir) {
-			initMetaDB()
-		}
-		if perfProfile {
-			go startPprofServer()
+		if currentCommand == "yb-voyager assess-migration-bulk" {
+			validateBulkAssessmentDirFlag()
+			if shouldLock(cmd) {
+				lockFPath := filepath.Join(bulkAssessmentDir, fmt.Sprintf(".%sLockfile.lck", GetCommandID(cmd)))
+				lockFile = lockfile.NewLockfile(lockFPath)
+				lockFile.Lock()
+			}
+			InitLogging(bulkAssessmentDir, cmd.Use == "status", GetCommandID(cmd))
+			startTime = time.Now()
+
+			if perfProfile {
+				go startPprofServer()
+			}
+			setControlPlane()
+		} else {
+			validateExportDirFlag()
+			schemaDir = filepath.Join(exportDir, "schema")
+			if shouldLock(cmd) {
+				lockFPath := filepath.Join(exportDir, fmt.Sprintf(".%sLockfile.lck", GetCommandID(cmd)))
+				lockFile = lockfile.NewLockfile(lockFPath)
+				lockFile.Lock()
+			}
+			InitLogging(exportDir, cmd.Use == "status", GetCommandID(cmd))
```

Comment:
yeah we can, but i don't much gain from doing that.
Also once we have call-home also implemented for this command, most of the code will be similar in both cases, then it will be easy to merge this code.